codec/decoder/plus/src/welsDecoderExt.cpp

```diff
@@ -581,6 +581,12 @@ DECODING_STATE CWelsDecoder::DecodeFrame2 (const unsigned char* kpSrc,
 
     eNalType = m_pDecContext->sCurNalHead.eNalUnitType;
 
+    if ((m_pDecContext->iErrorCode & dsRefLost) && m_pDecContext->eSliceType == B_SLICE) {
+      if (ResetDecoder())
```

Comment:
could you keep the coding style consistent by adding {} in this conditional? Thx